index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bhromaon</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="icon" href="favicon/favicon.ico">
+</head>
+
+<body>
+    <section class="header-wrapper">
+        <header class="header-wrapper__header">
+            <h1 class="title header-wrapper__header__title">Bhromaon</h1>
+            <p class="subtitle header-wrapper__header__subtitle">Safe Travel Anywhere</p>
+            <form class="form header-wrapper__header__form" action="#">
+                <input class="input form__input" type="search" name="Search"
+                    placeholder="Search Your Travel Destination">
+                <button class="button-submit form__button-submit">
+                    <i class="button-submit__icon icon-search"></i>Search
+                </button>
+            </form>
+            <div class="scroll-block header-wrapper__header__scroll-block">
+                <p class="scroll-block__text">Scroll Down</p>
+                <i class="scroll-block__icon icon-chevron-down"></i>
+            </div>
+        </header>
+    </section>
```

Comment:
написание классов вида `block__elem1__elem2` в БЭМ запрещено, потому что элемент может быть частью блока, а не другого элемента. Можно почитать [тут](https://ru.bem.info/methodology/quick-start/#%D0%AD%D0%BB%D0%B5%D0%BC%D0%B5%D0%BD%D1%82). Поэтому лучше сделать примерно так
```suggestion
    <header class="header">
        <div class="header__wrapper">
            <h1 class="title header__title">Bhromaon</h1>
            <p class="subtitle header__subtitle">Safe Travel Anywhere</p>
            <form class="form header__form" action="#">
                <input class="input form__input" type="search" name="Search"
                    placeholder="Search Your Travel Destination">
                <button class="button-submit form__button-submit">
                    <i class="button-submit__icon icon-search"></i>Search
                </button>
            </form>
            <div class="scroll-block header__scroll-block">
                <p class="scroll-block__text">Scroll Down</p>
                <i class="scroll-block__icon icon-chevron-down"></i>
            </div>
        </div>
    </header>
```
Просмотри похожие ошибки по коду.